pkg/handler/unrouted_handler.go

```diff
@@ -177,10 +177,10 @@ func (handler *UnroutedHandler) Middleware(h http.Handler) http.Handler {
 		// We also update the write deadline, but makes sure that it is larger than the read deadline, so we
 		// can still write a response in the case of a read timeout.
 		if err := c.resC.SetReadDeadline(time.Now().Add(handler.config.NetworkTimeout)); err != nil {
-			c.log.Warn("NetworkControlError", "error", err)
+			c.log.WarnCtx(c, "NetworkControlError", "error", err)
```

Comment:
Can you please use methods like `WarnContext` instead of `WarnCtx`? The latter ones are deprecated (see https://pkg.go.dev/golang.org/x/exp/slog#Logger.WarnCtx) and not part of the `log/slog` package in the standard library. In the next major release, we want to migrate from `golang.org/x/exp/slog` to `log/slog`, so this would be helpful.

Reply:
Yeah! Sorry about that - realize you said it in the issue, didn't realize that the `exp/slog` package had an update to provide it (earlier version only had `Ctx` variants). Updated in [3e95b11](https://github.com/tus/tusd/pull/1237/commits/3e95b1179960437a061ae0dc49551bd05183728c)!

Reply:
Ah, no worries! I didn't know what the currently used version of `slog` didn't have the newly named methods yet!